lib/pbench/server/api/__init__.py

```diff
@@ -12,11 +12,11 @@
 
 from pbench.server import PbenchServerConfig
 from pbench.common.exceptions import BadConfig
-from pbench.server.api.resources.elasticsearch_api import Elasticsearch
 from pbench.server.api.resources.upload_api import Upload, HostInfo
 from pbench.server.api.resources.graphql_api import GraphQL
-from pbench.server.api.resources.query_controllers import QueryControllers
 from pbench.common.logger import get_pbench_logger
+from pbench.server.api.resources.query_apis.elasticsearch_api import Elasticsearch
+from pbench.server.api.resources.query_apis.query_controllers import QueryControllers
```

Comment:
Any particular reason for this change?

Reply:
I just thought we could keep the elasticsearch related query apis in one place so its organized and easy to remove and/or add more apis or make changes that would reflect on all the es apis

Reply:
Typically we have been following the PEP 8 on [Imports](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
>
> Imports should be grouped in the following order:
>
> * Standard library imports.
> * Related third party imports.
> * Local application/library specific imports.
> You should put a blank line between each group of imports.